src/components/Contacts/ContactDetails/ContactDetailsTab/People/Items/EditPersonModal/EditPersonModal.tsx

```diff
@@ -44,80 +50,262 @@ const ShowExtraText = styled(Typography)(() => ({
   fontWeight: 'bold',
 }));
 
+const LoadingIndicator = styled(CircularProgress)(({ theme }) => ({
+  margin: theme.spacing(0, 1, 0, 0),
+}));
+
+const phoneRegex = RegExp(/^\(?([0-9]{3})\)?[-. ]?([0-9]{3})[-. ]?([0-9]{4})$/);
```

Comment:
Did you steal this from somewhere? I like putting a comment with the link if possible.

Reply:
Yeah, I grabbed it from StackOverflow while searching how to handle valid phone numbers. I'll go find it and add it as a comment.